### PR TITLE
Markup/calendar

### PR DESCRIPTION
**Pull request by jackiejakob**
Here's the start of the markup for the calendar.
Calendar is not being generated right now so moving between months is not functioning. 
But arrow keys and Home/End should be working. 

I have it starting with aria-current on 11/28 (today) and aria-selected on 11/14. 11/19 is an secondary selection indicator, but I'm thinking we add a way to define what this means in the component which can be added to the aria label. Thoughts?

@pauljadam 
VO is reading a "blank" in the label. Any idea why? I added aria-hidden to the text inside so it didn't read the number twice, but don't know what is causing the "blank".
screen shot 2018-11-28 at 3 47 10 pm

Any other feedback welcome!

### CODE REVIEW

File: calendar/markup-kit/js/main.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    calendars.forEach(calendar => {
+        const grid = calendar.querySelector(".pe-cal-dates"),
+            dates = calendar.querySelectorAll(".date-selector:not(:disabled)");
+
+        function unfocusAll() {
+            Array.prototype.forEach.call(dates, function(date) {
+                date.setAttribute("tabindex","-1");
+            });
+        }
+
```
[the remark this answers is not in the document]
Comment (jackiejakob) [reply calendar/markup-kit/js/main.js]: Let me know if these changes are what you envisioned. Thanks!

**Comment by davodey** [comment]
Im seeing this in IE11:

screen shot 2018-12-14 at 11 18 10 am

**Comment by davodey** [comment]
@jackiejakob I think this looks good.  Do you want to use Moment.JS to add calendar functionality?

**Comment by jackiejakob** [comment]
@davodey Good catch on IE. I'll take a look. I was thinking about using the Date object first before looking into Moment, unless you have experience that says otherwise?

**Comment by davodey** [comment]
@jackiejakob lets use Moment for this.  It should save a ton of time :)

https://momentjs.com/

CDN Link:
https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.2/moment.min.js

**Comment by antelopeb** [comment]
@jackiejakob, @davodey -- are we using moment.js for the component? Or are there more changes to be made to this?

**Comment by jackiejakob** [comment]
@antelopeb Yeah, I still need to fix the IE bug and integrate moment.js.

**Pull request merged**